plugins/strict-type-checker/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "blockly-plugin-strict-type-checker",
+  "version": "0.20200717.0",
```

Comment:
I think the create-package script never got updated after we switched to semver, so this should be 0.0.0 for now.

Filed https://github.com/google/blockly-samples/issues/334